rdm/0000-rdm-requests.md

```diff
@@ -0,0 +1,261 @@
+- Start Date: 2020-03-16
+- RFC PR: [#17](https://github.com/inveniosoftware/rfcs/pull/17)
+- Authors: Lars Holm Nielsen
+
```

Comment:
State? To comply with what was said in https://github.com/inveniosoftware/rfcs/issues/1